plugins/train/_config.py

```diff
@@ -68,23 +60,249 @@ def set_globals(self):
                  "for this initialization technique are expensive.")
         self.add_item(
             section=section, title="subpixel_upscaling", datatype=bool, default=False,
-            info="Use subpixel upscaling rather than pixel shuffler.\n"
-                 "Might increase speed at cost of VRAM")
+            info="\nUse subpixel upscaling rather than pixel shuffler. These techniques \n"
+                 "are both designed to produce better resolving upscaling than other \n"
+                 "methods. Each perform the same operations, but using different TF opts.\n"
+                 "https://arxiv.org/pdf/1609.05158.pdf \n")
         self.add_item(
             section=section, title="reflect_padding", datatype=bool, default=False,
-            info="Use reflect padding rather than zero padding. Only enable this option if the "
-                 "model you are training has a distinct line appearing around the edge of the "
-                 "swap area.")
-        self.add_item(
-            section=section, title="dssim_loss", datatype=bool, default=True,
-            info="Use DSSIM for Loss rather than Mean Absolute Error\n"
-                 "May increase overall quality.")
+            info="\nUse reflection padding rather than zero padding when either \n"
+                 "downscaling or using simple convolutions. Each convolution must \n"
+                 "pad the image/feature boundaries to maintain the proper sizing. \n"
+                 "More complex padding schemes can reduce artifacts at the border \n"
+                 "of the image.\n"
+                 "http://www-cs.engr.ccny.cuny.edu/~wolberg/cs470/hw/hw2_pad.txt \n")
         self.add_item(
             section=section, title="penalized_mask_loss", datatype=bool, default=True,
-            info="If using a mask, This penalizes the loss for the masked area, to give higher "
-                 "priority to the face area. \nShould increase overall quality and speed up "
-                 "training. This should probably be left at True")
-        logger.debug("Set global config")
+            info="\nImage loss function is weighted by mask presence. For areas of \n"
+                 "the image without the facial mask, reconstuction errors will be \n"
+                 "ignored while the masked face area is prioritized. May increase \n"
+                 "overall quality by focusing attention on the core face area.\n")
+        self.add_item(
+            section=section, title="image_loss_function", datatype=str,
+            default="Mean_Absolute_Error",
+            choices=["Mean_Absolute_Error", "Mean_Squared_Error", "LogCosh",
+                     "Smooth_L1", "L_inf_norm", "SSIM", "GMSD", "Pixel_Gradient_Difference"],
+            info="\nGeneral Loss Discussion \n"
```

Comment:
I admire the dedication to providing maximum possible information, but any attempt to make this slightly less verbose would be appreciated!

Reply:
I'll strip some -- Like I said earlier, I was fulfilling a request. Ha